fluss-server/src/test/java/com/alibaba/fluss/server/testutils/FlussClusterExtension.java

```diff
@@ -668,6 +668,21 @@ public Map<String, Long> waitUntilPartitionsCreated(TablePath tablePath, int exp
                 "Fail to wait " + expectCount + " partitions created");
     }
 
+    public Map<String, Long> waitUntilPartitionsDrop(TablePath tablePath, int expectCount) {
```

Comment:
The implementation of `waitUntilPartitionsDrop` is identical to `waitUntilPartitionsCreated`. The current implementation is not stable, as the partitions are created first and then dropped. It has chances that meets an intermediate state matches the expected partition count but the excepted expired partitions are not dropped. 

Therefore, would be better to add a `List<String> droppedPartitions` parameter, and assert the remaining partitions don't contain the `droppedPartitions`.